src/pages/thank-you-page/ui/OrderSummary/OrderSummary.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styles from './OrderSummary.module.scss';
+
+interface Item {
+  name: string;
+  price: number;
+  quantity: number;
+}
+
+interface Props {
+  items: Item[];
+  subtotal: number;
+  shipping: number;
+  total: number;
+}
+
+const OrderSummary: React.FC<Props> = ({
+  items,
+  subtotal,
+  shipping,
+  total,
+}) => (
+  <div className={styles.summary}>
+    <h3>
+      {/* eslint-disable-next-line react/jsx-one-expression-per-line */}
+      Your Order ({items.length})
+    </h3>
+    <ul className={styles.itemList}>
+      {items.map(item => (
+        <li key={item.name} className={styles.item}>
+          <div>
+            <p className={styles.itemName}>{item.name}</p>
+            <p className={styles.itemQuantity}>
+              {item.quantity}
```

Comment:
only price in the design
This needs to be clarified, do we need to use the number here?